### ISSUE TITLE
Suggestion - Basic Scope upgrade

### ISSUE THREAD

**Issue by DaemoniaV** [Heishe/idleraiders-second-run#342]
IMHO, basic scope is trash. Let it be more powerfull...
Damage +20
Crit Chance 12%
Crit Damage 130%
Attack Speed +12%
For example.

**Comment by TravisGesslein** [comment]
Why is it trash? It's stats are slightly better to a set item from the raids before (which is unique!). I.e. Ice Kiss has 13 damage and 15 attack speed, which should come out to a bit less DPS than the basic scope stats.

**Comment by DaemoniaV** [comment]
IMHO attack speed is very weak bonus cause main damage dealing is from Focused Shot and Quick Shot. IMHO Blackened Leather Armor much more power full - +3 CC and 100 CD it can be compare with +12 DAM and +7CC but +4 Focus Regen it is very powerfull. Two Armor twice my focus regen so grow dps very high. And finally HP  +250 never was redundand.

**Comment by TravisGesslein** [comment]
Mhm... what do you think about introducing some other items that give more energy regeneration? So you can combine some "Basic Scope"s with some of the new items for more energy regeneration? 

The itemization for different build options has many holes right now. For example, there are also few items for damage warriors (that increase rage gain, minimum/maximum rage gain) in various raids. Before the re-release we will spend a couple of days going over all the items in all the raids and try to fill holes in the itemization so you have many more options for different builds (as well as introducing more rares, uniques and artifacts, etc.).

**Comment by DaemoniaV** [comment]
Try to describe My view of item situation. My team after some lvl2 skills 90% clear Mountain Fefuge in Auto-Mode. All is My IMHO may be other testers have very another opinion.
All My WAR is 100% tanks. Two of them hold only shields and one jerk hold 4 set items. There is no other options. At point when last WAR wiped by bosses, other team fall in 60 sec. May be if You have total domination You prefer some DAM items for WAR to speed up clear time. But My 94 lvl team far away from this point.
Archers in My team DD and they have split 4 set items (2 for each) and 4 perefct Blackened Leather Armor.
Lets go more deeper. Main for DD is DAM. There is some ways to up the DAM.
1) Base DAM upgrade
2) CC (Crit Chance)
3) CD (Crit Damage)
4) FR (Focus Regeneration)
5) AS (Attack Speed)
Our main DD skills - QS and FS. For the 16 sec My archers do 7 simple shots, 4 QS and 2 FS. They almost hit Cool Down wall. But i can see that Focus never more 110. And there is every My archer have 1 FS and 1 QS book. If I can up FS more then I can equip more FS book. So the FR is very important stuff.
When I eqiup item? If his effect for Me seems more powerfull than old item.
If You add new generator item it need beat not scope but great Blackened Leather Armor. But in that point I prefer two new items without scope. You can do the small set Scope and Silver Arrow when both equip do some FS but more +8.
There is We go to the some bothering Me. There is slot restricting. At this point every Raider have 4 slots. If we suppose that it is free slots it is enought. But set is very powerfull so they must have. At current point.... 40 slots and 16 will be "block" by sets. And with some new Raid there will be break point of painfull choose between new single items and set items.
There is some ways to do it less pain)))
Simple - 1) For clearing some new Raid add one equipment slot
Complicated - 2) When new Raid release thre is new version for all old set items with more powerfull stats, But it can be dropped only form last aviable Raid.

**Comment by TravisGesslein** [comment]
Sorry for not getting around to responding to this. I read it of course, and it helped influence some design decisions for the future. There's another issue, #415 that is related to this (so I will close this issue for now). 

One of the things we want to do is have item sets be smaller, or the required number of items equipped be smaller to unlock all the set bonuses (we implemented this with the craftable upgraded versions of the Sealer set that will be coming in the first post released patch). This should reduce the 'slot pressure' that set items have significantly. 

Adding more equipment slots is another way, of course, but we reduced those in 2nd run specifically because we wanted to reduce the amount of micro managing (lots of clicking) that players had to do. 

Also the stat suggestions are taken into account, of course. We'll be looking to provide more item diversity in the future :)
